### ISSUE TITLE
Unrecoverable error: undefined in DELAYBIND

### ISSUE THREAD

**Issue by tuxedo0801** [pmonta/gerber2graphtec#16]
The g2g worked like a charm on my debian testing so far.
Today, I wanted to plot another stencil and got this output for g2g_gui on the console:

```bash
 *** WARNING - the DELAYBIND command has been deprecated, and will be
               removed in the next version. For now you can restore
               the behaviour by using -dREALLYDEALYBIND but if you
               require continued use of this command you should contact
               the Ghostscript developers. Commercial customers of
               Artifex should email their support contact, free users
               are encouraged to talk to us on the #ghostscript IRC
               channel on irc.freenode.net.

Unrecoverable error: undefined in DELAYBIND
PostScript/PDF Interpreter finished. Return status 65280 executed command : /usr/bin/gs -q -dDELAYBIND -dWRITESYSTEMDICT -dNODISPLAY -dNOEPS "/tmp/psinJYnEC4"
The interpreter seems to have failed, cannot proceed !
```

Seems that something has changed on my system. Any ideas how to fix?

**Comment by tuxedo0801** [comment]
Seems the error comes from pstoedit. Currently I have:

```
apt policy pstoedit
pstoedit:
  Installiert:           3.70-5
  Installationskandidat: 3.70-5
  Versionstabelle:
 *** 3.70-5 500
        500 http://ftp.de.debian.org/debian testing/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by tuxedo0801** [comment]
Reproduced the issue by invokin manually these commands:

gerbv --export=pdf --output=test.pdf --border=0 mystencil.GTP
pstoedit -q -f pic test.pdf test.pic

Once pstoedit is invoked, the error occurs. Seems that pstoedit has a new/changed behavior.

It's unclear to me how to overcome this issue. 

Will further investigate ...

**Comment by tuxedo0801** [comment]
... pstoedit is calling ghostscript, which then causes this error. It's somehow related to 

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=862779
https://bugs.ghostscript.com/show_bug.cgi?id=697846

No idea how to fix this issue. downgrade is somehow a nasty workaround.

**Comment by BartMassey** [comment]
Solution is to fix `gerbv` to not emit `.bind` in its Postscript, I imagine. I'm not enough of a Postscript guru to supply a patch.

**Comment by tuxedo0801** [comment]
gerbv produces .pdf, not .ps, and pstoedit takes .pdf as input.

Or do you mean the pdf-tags/commands produced by gerbv are kind of ".bind" ?!

Think I'm even less a ps guru than you :-)

**Comment by BartMassey** [comment]
Yeah, sorry, PDF/Postscript. It's all kind of the same thing under the hood. Anyway, looks to me like the problem is noncomformant Postscript produced by `gerbv` that then doesn't get processed properly by `pstoedit` because of the recent security fix in Ghostscript. I haven't investigated, and may be wrong.

**Comment by tuxedo0801** [comment]
maybe I downgrade ghostscript. But actually I don't want to screw my system.

Does anyone know about another solution to create pic from pdf?

**Comment by tuxedo0801** [comment]
Downgrade-Solution:

https://github.com/pmonta/gerber2graphtec/issues/15#issuecomment-357777922

**Comment by srdco** [comment]
Also, can now upgrade pstoedit 3.71 and use -rdb flag.
Just figured this out today and posted on https://askubuntu.com/questions/1034894/render-latex-is-not-working-in-inkscape-after-installing-to-18-04.
